c-cannons/bangazon-initial_site
# DevOps

### Pull Requests

When you submit a pull request to the project repository, it should provide all of the information necessary for one of your teammates to verify its completeness.

#### Descriptions

The description that you provide should be comprehensive enough to...

1. Provide clarity to any potentially complex code.
1. Explain reasons behind organizational or architectual decisions you made.
1. Give context to what feature you were completing so that your teammate has a mental model before looking at the code.

#### Steps to Test

You must provide clear steps for any teammate to test the code.

1. System configuration.
1. 3rd party libraries that need to be installed.
1. Command line utilities to run.
1. If there is a UI component, give clear instructions for steps to perform in the UI, and what they should expect to see as the outcome of those steps.

#### Link to Feature Ticket

At the end of the PR description, you must provide a hyperlink to the ticket that contains a description of the feature you are working on.
